presto-docs/src/main/sphinx/release/release-0.115.rst

```diff
@@ -8,3 +8,9 @@ Hive Changes
 * Fix a race condition which could cause queries to finish without reading all the data.
 * Fix a bug in Parquet reader that causes failures while reading lists that has an element
   schema name other than ``array_element`` in its Parquet-level schema.
+
+General Changes
+---------------
+
+* Fix issue in query queues where queries could be rejected after being accepted,
```

Comment:
How about "Fix an issue with hierarchical queue rules where queries could be rejected after being accepted."